src/style.css

```diff
@@ -1,3 +1,91 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). It's best to pick one direction (either top or bottom) for vertical margins to prevent issues with margin collapsing and to maintain consistency throughout the stylesheet.